platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -64,16 +64,6 @@ typedef NS_ENUM(NSUInteger, MGLCircleTranslationAnchor) {
  ### Example
 
  ```swift
- let layer = MGLCircleStyleLayer(identifier: "circles", source: population)
```

Comment:
Any idea why this code went missing? It’s present on both master and the release branch? Same for all the other style layer headers.